src/HTML5/Parser/UTF8Utils.php

```diff
@@ -48,9 +48,7 @@ class UTF8Utils
     public static function countChars($string)
     {
         // Get the length for the string we need.
-        if (function_exists('iconv_strlen')) {
-            return iconv_strlen($string, 'utf-8');
-        } elseif (function_exists('mb_strlen')) {
```

Comment:
I'd like to have as wide of support as possible. Instead of removing `iconv_strlen` all together can you re-order them so `mb_strlen` is tried first. Then, if `mb_strlen` isn't present it can still try `iconv_strlen`.